### DISCUSSION TITLE
only edit SelectColumn if condition is met

### DISCUSSION THREAD

**Discussion by abdikafar** [filamentphp/filament#6772] [Help] [answered]
I have the following SelectColumn `Tables\Columns\SelectColumn::make('payment_method_id')
                    ->options(
                        PaymentMethod::all()
                            ->pluck('name', 'id')
                            ->toArray(),
                    )
                    ->label('Payment Method'),` only make this column editable if the status of Transaction is 'Pending'. how to acheive this?

**Comment by pboivin** [comment, marked as answer]
Maybe something like this?
```php
    ->disabled(fn ($record) => $record->status !== 'pending')
```

**Reply by abdikafar** [reply]
Thanks. tested it and it worked.
